services/payments/handler.js

```diff
@@ -300,22 +300,22 @@ export const webhook = async (event, ctx, callback) => {
     }
 
     switch (data.paymentType) {
-      case "UserMember":
-        await userMemberSignup(data);
-        break;
-      case "OAuthMember":
-        await OAuthMemberSignup(data);
-        break;
-      case "Member":
-        await memberSignup(data);
-        break;
-      case "Event":
-        await eventRegistration(data);
-        break;
-      default:
-        return helpers.createResponse(400, {
-          message: "Webhook Error: unidentified payment type"
-        });
+    case "UserMember":
+      await userMemberSignup(data);
+      break;
+    case "OAuthMember":
+      await OAuthMemberSignup(data);
+      break;
+    case "Member":
```

Comment:
just making sure these are codebase eslint changes and not personal styler overriding them, can you do an npm run lint